packages/jest-react/src/internalAct.js

```diff
@@ -66,20 +66,21 @@ export function act(scope: () => Thenable<mixed> | void) {
   // returned and 2) we could use async/await. Since it's only our used in
   // our test suite, we should be able to.
   try {
-    const thenable = scope();
+    const result = scope();
```

Comment:
I updated our internal implementation of `act` to resolve to the return value of the scope function, like how the public implementation of `act` works. Not relevant to the rest of the PR, just wanted it for a test.